.github/workflows/functional-tests.yml

```
@@ -29,5 +29,15 @@ jobs:
         find src -name spec.yaml -print0 | xargs -0 -n1 aac validate
         find src/aac/plugins -name *.yaml -print0 | xargs -0 -n1 aac validate
 
+    - name: Validate Example Model
+      run: |
+        aac validate model/flow/System.yaml
+
+    - name: Test gen-plugin
+      run: |
+        mkdir -p AaC/model/test
+        cp AaC/plugins/aac-spec/aac-spec.yaml AaC/model/test
+        aac gen-plugin AaC/model/test/aac-spec.yaml
```

This should address the pipeline issue. I also suggest changing the temporary directory name to be verbose rather than adding complexity to the directory structure via multiple levels.
```suggestion
        mkdir gen_plugin_generation_test
        cp  src/aac/spec/spec.yaml gen_plugin_generation_test
        aac gen-plugin gen_plugin_generation_test/spec.yaml
```

---

Thanks again